### ISSUE TITLE
Improve some docstrings

### ISSUE THREAD

**Issue by natema** [worlddynamics/WorldDynamics.jl#177]
https://github.com/worlddynamics/WorldDynamics.jl/blob/1ea057e847a170bb38a1726b7e872f9c3fc7b9ae/src/functions.jl#L9

- What about enforcing one-line-per-sentence in docstrings?
- The definition of interpolate is unclear and can be improved, in particular when we say `by linearly interpolating the function itself through the table`.

**Comment by paulobruno** [comment]
> What about enforcing one-line-per-sentence in docstrings?

The common approach is to enforce at most 80 characters. I'm fine with 80 chars.

> The docstring of the Interpolate single value is missing

Do we need it? It's no supposed to be public. 
Maybe adding an underscore to follow the common "private" functions pattern. What do you think?

**Comment by natema** [comment]
I agree with underscore with no docstrings.
I'm also fine with the 80 char limit if that's more common, even though one-sentence-per-line should be easier to diff.

**Comment by paulobruno** [comment]
Yes, one sentence per line is better for diff, which is very useful for text files, like readme and latex.

**Comment by natema** [comment]
Does it mean you agree with adopting that convention in docstrings?

**Comment by paulobruno** [comment]
The case of code and docstrings is that some things, like old terminal versions and Julia documentation renderer, trim the text.
In this case, we should go for the 80 chars limitation.

**Comment by natema** [comment]
Julia doesn't trim the text. 
I don't know which terminal still used today do.

**Comment by natema** [comment]
Ah sorry I read it too fast, you mean code blocks displayed in the documentation pages... right.
